src/es.json

```diff
@@ -251,37 +251,37 @@
             "AlignTop": "Alinear la parte superior",
             "AlignMiddle": "Alinear Medio",
             "AlignBottom": "Alinear la parte inferior",
-            "MergeCells": "Combinar células",
+            "MergeCells": "Combinar celdas",
             "MergeAll": "Fusionar todo",
             "MergeHorizontally": "Fusionar horizontalmente",
             "MergeVertically": "Fusionar verticalmente",
             "Unmerge": "Desunir",
-            "UnmergeCells": "Unmerge Cells",
-            "SelectMergeType": "Seleccionar tipo de fusión",
-            "MergeCellsAlert": "La combinación de celdas solo conservará el valor superior izquierdo (superior). Fusionar de todos modos?",
+            "UnmergeCells": "Desunir Celdas",
+            "SelectMergeType": "Seleccionar tipo de combinación",
```

Comment:
Use this "Seleccione el tipo de combinación"

Reply:
Changed